CHANGELOG.md

```diff
@@ -2,13 +2,101 @@
 
 ## Unreleased
 
+## v0.120.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.120.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.120.0) and the [opentelemetry-collector-contrib v0.120.1](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.120.1) releases where appropriate.
+
 ### 🛑 Breaking changes 🛑
 
-- (all) Bump minimum go version to 1.23 ([#5920](https://github.com/signalfx/splunk-otel-collector/pull/5920))
+- (Contrib) `receiver/prometheus`: Prometheus receiver now uses scrapers in Prometheus 3.0. ([#36873](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36873))
+  There are a number of breaking changes in Prometheus 3.0. Learn more about those changes and migration guide on https://prometheus.io/docs/prometheus/latest/migration/.
+- (Contrib) `activedirectorydsreceiver`: Fixed typo in the attribute `distingushed_names`, renaming it to `distinguished_names`. ([#37606](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37606))
+- (Contrib) `receiver/hostmetrics`: Remove receiver.hostmetrics.normalizeProcessCPUUtilization feature gate ([#34763](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34763))
+- (Contrib) `tailsamplingprocessor`: Fix the decision timer metric to capture longer latencies beyond 50ms. ([#37722](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37722))
+  This changes the unit of the decision timer metric from microseconds to milliseconds.
+- (Contrib) `routingconnector`: Remove `match_once` configuration parameter. ([#36824](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36824))
+- (Core) `service`: Align component logger attributes with those defined in RFC ([#12217](https://github.com/open-telemetry/opentelemetry-collector/pull/12217))
+  See [Pipeline Component Telemetry RFC](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/rfcs/component-universal-telemetry.md#attributes)
+
+### 💡 Enhancements 💡
+
+- (Splunk) Make Windows TA agnostic to Powershell ExecutionPolicy ([#5935](https://github.com/signalfx/splunk-otel-collector/pull/5935))
+- (Splunk) Bump pygithub from 2.6.0 to 2.6.1 in /packaging/release ([#5933](https://github.com/signalfx/splunk-otel-collector/pull/5933))
+- (Splunk) Bump boto3 from 1.36.21 to 1.36.23 in /packaging/release ([#5923](https://github.com/signalfx/splunk-otel-collector/pull/5923))
+- (Splunk) Bump go.etcd.io/bbolt from 1.3.11 to 1.4.0 ([#5881](https://github.com/signalfx/splunk-otel-collector/pull/5881))
```

Comment:
Still should be removed

Reply:
Gotcha. Done. I really thought these could affected customers.